articles/data-explorer/ingest-data-streaming.md

```diff
@@ -68,8 +68,7 @@ There are two supported streaming ingestion types:
 
 ## Limitations
 
-* Streaming ingestion performance and capacity scales with increased VM and cluster sizes. For a single D14 node, the recommended load is up to 150 requests per second.
-* Currently, support is only for 8 and 16 core SKUs (D13, D14, L8, and L16).
+* Streaming ingestion performance and capacity scales with increased VM and cluster sizes. Concurrent ingestions are limited to 6 ingestions per core. For example, for 16 cores SKUs (such as D14 and L16) the maximal supported load is 96 concurrent ingestions and for 2 cores SKUs (such ad D11) the maximum is 12.
```

Comment:
For example, for 16 core SKUs, such as D14 and L16, the maximal supported load is 96 concurrent ingestions. For 2 core SKUs, such as D11, the maximal supported load is 12 concurrent ingestions.

Reply:
Please update that section with edited sentences.